Source/Core/DolphinQt2/QtUtils/AddConnectedAction.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QAction>
+#include <QObject>
+#include <QString>
+#include <QWidget>
+
+// Qt5 added a new, compile-time way to connect to signals (passing functors or member function
+// pointers directly), but QMenu's addAction method didn't get equivalent functionality until Qt
+// 5.6. These functions are meant to be drop-in replacements for the new QMenu::addAction overrides
+// until Dolphin requires at least Qt 5.6 to build.
```

Comment:
We did, but Ubuntu LTS only has Qt 5.5 (and Debian Jessie only has 5.3, though it's been now superseded by Stretch). In the interest of allowing users to easily compile and getting the buildbots up-and-running, I figured it's best to keep the requirements low for now.

This should be a quick replace if we come to a decision later to only allow >=5.6.